crates/diman_unit_system/src/codegen/float_methods.rs

```diff
@@ -26,6 +38,7 @@ impl Defs {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
I'd change this to `#[cfg(any(feature = "std", feature = "num-traits-libm"))]`

Reply:
Then it also needs to be applied transitively to `dimensionless_float_method` (above). I don't have a strong preference, but I tend to prefer having the compiler type-check those functions in all configurations.

Reply:
Oh, right. What about
`#[cfg_attr(not(any(feature = "std", feature = "num-traits-libm")), allow(dead_code))]`?
Looks a little ugly but should do both.

Reply:
Great, will do.